src/AutoBalance.cpp

```diff
@@ -459,6 +459,21 @@ class AutoBalance_AllCreatureScript : public AllCreatureScript
 
         CreatureTemplate const *creatureTemplate = creature->GetCreatureTemplate();
 
+		uint32 mapId = creature->GetMapId();
+		if (mapId == 30 || mapId == 628)
+			return;
+		
+        uint32 maxNumberOfPlayers = ((InstanceMap*)sMapMgr->FindMap(creature->GetMapId(), creature->GetInstanceId()))->GetMaxPlayers();
+        //    SOLO  - By MobID
+        if (GetForcedCreatureId(creatureTemplate->Entry) >= 0)
```

Comment:
check the array, okay, empty return -1
We get to the check there -1, and that's more or equal to 0, so we pass by.